### PR TITLE
Theme: TitlePane

### PR DESCRIPTION
**Pull request by bitpshr**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**

This PR adds an initial theme for the TitlePane component.

image

### CODE REVIEW

File: src/themes/dojo/titlePane.m.css [hunk 1]
```diff
@@ -1,30 +1,55 @@
...
+.title > div {
+	background-color: var(--color-background);
+	border: var(--border-width) solid var(--color-border);
+	color: var(--color-text-faded);
+	padding: var(--grid-base) var(--grid-base) var(--grid-base) calc(var(--grid-base) * 4);
 	position: relative;
 	z-index: 1;
 }
 
+.title > div:focus {
```
Comment (tomdye) [inline src/themes/dojo/titlePane.m.css]: is there a class that can be used instead of `div` ?

File: src/themes/dojo/titlePane.m.css [hunk 2]
```diff
@@ -1,30 +1,55 @@
...
+	padding: var(--grid-base);
+	transition: margin-top ease-in-out var(--transition-duration);
+}
+
+.open .title > div {
+	color: var(--color-highlight);
+}
+
+.arrow {
+	position: absolute;
```
Comment (tomdye) [inline src/themes/dojo/titlePane.m.css]: Can positioning etc... be moved to a fixed class?

Comment (bitpshr) [inline src/themes/dojo/titlePane.m.css]: I think we can definitely make better use of fixed classes (I am working on that now) but I don't think stylistic rules that aren't necessitated for usability like positioning of icons should be in fixed classes. To me, that's purely themeable stuff.

File: src/themes/dojo/titlePane.m.css (hunk 1 above)
Comment (bitpshr) [inline src/themes/dojo/titlePane.m.css]: Good catch. Fixed.

**Comment by bitpshr** [comment]
Thanks @tomdye. I used a class instead of the element selector. Should be ready for another review.

**Pull request merged**